src/Avalonia.Wayland/WaylandImmediateRendererProxy.cs

```diff
@@ -0,0 +1,112 @@
+using Avalonia.Rendering;
+using Avalonia.Threading;
+using Avalonia.VisualTree;
+
+namespace Avalonia.Wayland
+{
+    public class WaylandImmediateRendererProxy : IRenderer, IRenderLoopTask
```

Comment:
This should probably be extracted to `RenderLoopImmediateRenderer` since it's a copy-paste

Reply:
👍